arch/arm/core/aarch64/reset.S

```diff
@@ -119,7 +118,7 @@ SECTION_SUBSEC_FUNC(TEXT,_reset_section,__start)
 	adr	x0, 1f
 	msr	elr_el3, x0
 	eret
-#endif
+
 	/*
 	 * Enable the instruction cache, stack pointer and data access
 	 * alignment checks.
```

Comment:
EL2 code from `reset.S` also should be removed?

Reply:
In theory you could start QEMU in EL2 and drop to EL1 from there. So I'd leave that in place.

Reply:
Uhm, actually I just realized that from EL2 we are not dropping to EL1. At this point I'd just deprecate EL2 and remove also that code path.